app/views/devise/sessions/new.html.erb

```diff
@@ -1,22 +1,69 @@
-<div class="panel panel-default">
-  <div class="panel-heading">
-    <h1 class='panel-title'>Sign In</h1>
-  </div>
+<div class="splash-panel">
+  <div class="splash-title-container">
+    <h1>cypress</h1>
+    <h2>Rigorouts &amp; repeatable testing of Electronic Health Records</h2>
+    <p>Powered by Super Duper Testing Labs</p>
```

Comment:
Yeah that shouldn't be hardcoded, I'll remove it in my commit. I'll get feedback from the ATLs and see if what they want customized. We can make that it's own feature in Pivotal as there might be other customization they want.